src/hooks/index.js

```diff
@@ -432,49 +511,59 @@ exports.restrictToCurrentSchool = async (context) => {
 /* todo: Many request pass id as second parameter, but it is confused with the logic that should pass.
 	It should evaluate and make clearly.
  */
-const userIsInThatCourse = (user, { userIds = [], teacherIds = [], substitutionIds = [] }, isCourse) => {
+const userIsInThatCourse = (
+	user,
+	{ userIds = [], teacherIds = [], substitutionIds = [] },
+	isCourse,
+) => {
 	const userId = user._id.toString();
 	if (isCourse) {
-		return userIds.some((u) => equalIds(u, userId))
-			|| teacherIds.some((u) => equalIds(u, userId))
-			|| substitutionIds.some((u) => equalIds(u, userId));
+		return (
+			userIds.some((u) => equalIds(u, userId)) ||
+			teacherIds.some((u) => equalIds(u, userId)) ||
+			substitutionIds.some((u) => equalIds(u, userId))
+		);
 	}
 
-	return userIds.some((u) => equalIds(u, userId)) || testIfRoleNameExist(user, 'teacher');
+	return (
+		userIds.some((u) => equalIds(u, userId)) ||
+		testIfRoleNameExist(user, 'teacher')
+	);
 };
 
-exports.restrictToUsersOwnCourses = (context) => getUser(context).then((user) => {
-	if (testIfRoleNameExist(user, ['superhero', 'administrator'])) {
-		return context;
-	}
-	const { _id } = user;
-	if (context.method === 'find') {
-		context.params.query.$and = (context.params.query.$and || []);
-		context.params.query.$and.push({
-			$or: [
-				{ userIds: _id },
-				{ teacherIds: _id },
-				{ substitutionIds: _id },
-			],
-		});
-	} else {
-		const courseService = context.app.service('courses');
-		return courseService.get(context.id).then((course) => {
-			if (!userIsInThatCourse(user, course, true)) {
-				throw new Forbidden('You are not in that course.');
-			}
-		});
-	}
+exports.restrictToUsersOwnCourses = (context) =>
+	getUser(context).then((user) => {
+		if (testIfRoleNameExist(user, ['superhero', 'administrator'])) {
+			return context;
+		}
+		const { _id } = user;
+		if (context.method === 'find') {
+			context.params.query.$and = context.params.query.$and || [];
+			context.params.query.$and.push({
+				$or: [
+					{ userIds: _id },
+					{ teacherIds: _id },
+					{ substitutionIds: _id },
+				],
+			});
+		} else {
+			const courseService = context.app.service('courses');
+			return courseService.get(context.id).then((course) => {
+				if (!userIsInThatCourse(user, course, true)) {
+					throw new Forbidden('You are not in that course.');
+				}
+			});
+		}
 
-	return context;
-});
+		return context;
+	});
 
 const isProductionMode = NODE_ENV === ENVIRONMENTS.PRODUCTION;
 exports.mapPayload = (context) => {
 	if (!isProductionMode) {
 		logger.info(
-			'DEPRECATED: mapPayload hook should be used to ensure backwards compatibility only, '
-			+ `and be removed if possible. path: ${context.path} method: ${context.method}`,
+			'DEPRECATED: mapPayload hook should be used to ensure backwards compatibility only, ' +
```

Comment:
Codacy found an issue: ['+' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)